config.go

```diff
@@ -17,8 +17,7 @@ type Config struct { // nolint: aligncheck
 	Linters map[string]string
 
 	// The set of linters that should be enabled.
-	Enable  []string
-	Disable []string
```

Comment:
Yes, I use disable flag too, but actually all linters user wants to disable [are deleted](https://github.com/alecthomas/gometalinter/blob/master/main.go#L191) from config.Enable